bundle-workflow/test.sh

```diff
@@ -9,4 +9,11 @@
 set -e
 
 DIR="$(dirname "$0")"
-"$DIR/run.sh" "$DIR/src/test.py" $@
+case $1 in
```

Comment:
This change is also present in #456 now as requested. Can we remove it from here?

Reply:
Sure that works :)